src/languages/en-US.ts

```diff
@@ -3630,6 +3630,7 @@ export default class extends Language {
 		COMMAND_SUGGEST_NOSETUP: username => `I'm sorry ${username}, but a suggestions channel hasn't been set up.`,
 		COMMAND_SUGGEST_NOSETUP_ASK: username => `I'm sorry ${username}, but a suggestions channel hasn't been set up. Would you like to set up a channel now?`,
 		COMMAND_SUGGEST_NOSETUP_ABORT: 'Alright then. Aborted creating a new suggestion.',
+		COMMAND_SUGGEST_NOPERMISSIONS: (username, channel) => `I'm sorry ${username}, but the administrators didn't give me the permissions to send messages in ${channel}!`,
```

Comment:
I feel this reads a bit better:
```suggestion
		COMMAND_SUGGEST_NOPERMISSIONS: (username, channel) => `I'm sorry ${username}, but the administrators didn't give me permission to send messages in ${channel}!`,
```